infrastructure/helm/keycloak/configure-keycloak.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
```

Comment:
I would say that we can assume that the user can modify these values from `values.yaml` by itself and remove this script, what do you think @akihikokuroda ? Like the rest of the values.

Reply:
I feel that it may be a little difficult for some users who doesn't have knowledge of cluster.  I probably documents the contents of script in the doc instead of putting the script file here.

Reply:
My comment comes thinking in if someday we publish this helm chart the installation should be as much as "standard" as we can. We can expect some configuration from the user but I think that the execution of a script is not quite common (feel free to correct me if I'm wrong).

Reply:
Yes, I agree.  I had to reinstall a lot of times so scripts helped me a lot but instructions in the README are probably enough for the user who installs only once.